### PR TITLE
added support to new display and lvgl

### PR DESCRIPTION
**Pull request by Rocketct**
- added support to display with GC9A01 Driver
- added lv_conf.h file to allows for lvgl configuration

for future example where would be used lvgl, add on top headers inclusion the lv_conf.h file

### CODE REVIEW

File: src/Arduino_MKRIoTCarrier.h [hunk 1]
```diff
@@ -118,10 +111,11 @@ class MKRIoTCarrier{
     MKRIoTCarrierQtouch Button4 __attribute__((deprecated)) = MKRIoTCarrierQtouch(TOUCH4);
 
     //Display
-    Adafruit_ST7789 display = Adafruit_ST7789(&SPI, TFT_CS, TFT_DC, -1);
-	
+    MKRIoTDispalyClass display = MKRIoTDispalyClass();
```
Comment (alranel) [inline src/Arduino_MKRIoTCarrier.h:114]: There's a typo: "Dispaly" should be "Display"

**Review by alranel — CHANGES_REQUESTED** [review]

**Review by marqdevx — APPROVED** [review]
LGTM!

**Comment by aentinger** [comment]
@Rocketct - as far as I can see, we can merge this one, no?

**Comment by Rocketct** [comment]
ciao Alex unfortunately no, even if this is fine, we shall wait that the new display driver is confirmed

**Comment by aentinger** [comment]
@Rocketct any update if this has been validated with the new display driver?

**Pull request closed without merging**